public/javascripts/SVLabel/src/SVLabel/onboarding/Onboarding.js

```diff
@@ -726,6 +735,45 @@ function Onboarding (svl, actionStack, audioEffect, compass, form, handAnimation
         $target.on("click", callback);
     }
 
+    /**
+     * Tell the user to label the multiple target attributes.
+     * @param state
+     * @param listener
+     * @private
+     */
+    function _visitLabelMultipleAccessibilityAttributeState(state, listener) {
+
+        var $target = uiCanvas.drawingLayer;
+
+        var callback = function (e) {
+
+            var st = state;
+            var i = 0;
+
+            while(i < st.properties.length){
+                var imageX = st.properties[i].imageX;
+                var imageY = st.properties[i].imageY;
+                var tolerance = st.properties[i].tolerance;
+
+                var clickCoordinate = mouseposition(e, this),
```

Comment:
This code seems to be repeated here from _visitLabelAccessibilityAttributeState. Could you create a common function and use it it both places.